localstack/entrypoint/init-aws.sh

```diff
@@ -13,56 +13,3 @@ for email in "${VALID_EMAILS[@]}"; do
 done
 
 awslocal s3api create-bucket --bucket cpf-reporter --region us-west-2 --create-bucket-configuration '{"LocationConstraint": "us-west-2"}'
-
```

Comment:
is this deletion intended?

Reply:
It is. I tend to removing trailing whitespace from the end of files. I can add it back in if you prefer.

Reply:
apologies I didn't intend to select this specific line was referring to lines 16-68. Will create a separate thread.